tests/e2e/vm-secret-sync/01-update-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+unitTest: false
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: example-secret
+data:
+  foo: Z29vZGJ5ZSB3b3JsZA==
```

Comment:
suggestion: Add a comment saying what the value is? (I guess also in `00-create-vm.yaml` as well)